modin/data_management/functions/default_methods/groupby_default.py

```diff
@@ -232,10 +284,19 @@ def fn(
             if not as_index:
                 if (
                     len(result.index.names) == 1 and result.index.names[0] is None
-                ) or all([name in result.columns for name in result.index.names]):
+                ) or all(name in result.columns.values for name in result.index.names):
```

Comment:
Why is this extra call (`.values`) needed?

Reply:
outdated